.coveragerc

```diff
@@ -1,11 +1,7 @@
 [run]
 branch = True
 parallel = True
-source = synapse
-
-[paths]
-source=
-   coverage
```

Comment:
What does this do?

Reply:
fixes our coverage to actually cover the storage APIs :(